bazel/pipeline.bzl

```diff
@@ -66,25 +70,41 @@ def sematic_pipeline(
     if dev:
         main = "@sematic//sematic/resolvers:worker.py"
         srcs = ["@sematic//sematic/resolvers:worker.py"]
+        script_data = ["@sematic//bazel:ray", "@sematic//bazel:bazel_python"]
```

Comment:
Add `bazel_python` to images built with this macro

Reply:
What are the levers for users to switch on or off having ray in the image? Is it an explicit dependency? Something else?